diff --git a/code/game/bg_pmove.c b/code/game/bg_pmove.c
index 5465c579..c981c954 100644
--- a/code/game/bg_pmove.c
+++ b/code/game/bg_pmove.c
@@ -1634,55 +1634,50 @@ static void PM_Weapon( void ) {
 		return;
 	}
 
-	// take an ammo away if not infinite
-	if ( pm->ps->ammo[ pm->ps->weapon ] != -1 ) {
-		pm->ps->ammo[ pm->ps->weapon ]--;
-	}
-
 	// fire weapon
 	PM_AddEvent( EV_FIRE_WEAPON );
 
 	switch( pm->ps->weapon ) {
 	default:
 	case WP_GAUNTLET:
-		addTime = 400;
+		addTime = 0;
 		break;
 	case WP_LIGHTNING:
-		addTime = 50;
+		addTime = 0;
 		break;
 	case WP_SHOTGUN:
-		addTime = 1000;
+		addTime = 0;
 		break;
 	case WP_MACHINEGUN:
-		addTime = 100;
+		addTime = 0;
 		break;
 	case WP_GRENADE_LAUNCHER:
-		addTime = 800;
+		addTime = 0;
 		break;
 	case WP_ROCKET_LAUNCHER:
-		addTime = 800;
+		addTime = 0;
 		break;
 	case WP_PLASMAGUN:
-		addTime = 100;
+		addTime = 0;
 		break;
 	case WP_RAILGUN:
-		addTime = 1500;
+		addTime = 0;
 		break;
 	case WP_BFG:
-		addTime = 200;
+		addTime = 0;
 		break;
 	case WP_GRAPPLING_HOOK:
-		addTime = 400;
+		addTime = 0;
 		break;
 #ifdef MISSIONPACK
 	case WP_NAILGUN:
-		addTime = 1000;
+		addTime = 0;
 		break;
 	case WP_PROX_LAUNCHER:
-		addTime = 800;
+		addTime = 0;
 		break;
 	case WP_CHAINGUN:
-		addTime = 30;
+		addTime = 0;
 		break;
 #endif
 	}
